pkg/ccn-lite/Makefile

```diff
@@ -13,11 +16,22 @@ all: $(PKG_BUILDDIR)/src/Makefile
 	$(MAKE) -C $(PKG_BUILDDIR)/src && \
 	cp $(PKG_BUILDDIR)/src/lib/libccnl-riot.a $(BINDIR)/ccn-lite.a
 
-$(PKG_BUILDDIR)/src/Makefile: $(TOOLCHAIN_FILE)
+$(PKG_BUILDDIR)/src/Makefile: $(TOOLCHAIN_FILE) | ..cmake_version_supported
```

Comment:
This is executed with every run without being cached, right? Can't we touch a file to skip subsequent version checks for an existing binary dir?

Reply:
Here `TOOLCHAIN_FILE` is also rebuilt everytime as it depends on `git-download` and so is `Makefile` created by `cmake`…

A solution would still require to save `command -v cmake` to a file and verify it did not change since last call.
So I would more do it after I can finish working on https://github.com/RIOT-OS/RIOT/pull/9634.

Reply:
can't you run the cmake version check *before* downloading or generating the toolchain file?

Reply:
I was more concentrated on getting a working test than where I should put it.

Putting it as a check before downloading could be problematic if `download` gets done at a separate time or even a different environment. I still hope for a separate "prepare" step. But it could be moved again later.

So yes I can move it.